src/text_embedding.rs

```diff
@@ -138,14 +141,23 @@ impl TextEmbedding {
                 .expect("Failed to retrieve model.onnx_data.");
         }
 
+        // prioritise loading pooling config if available, if not (thanks qdrant!), look for it in hardcoded
+        let post_processing = match model_repo.get("1_Pooling/config.json") {
+            Err(_) => model_name.get_default_pooling_method(),
```

Comment:
Hey @triandco.
If we're specifying the pooling strategy for each model in `get_default_pooling_method()`, is trying to read the config file needed at all?

Reply:
the default method is only a fall back. Ideally, we should always get it from the source in online mode, since it can get updated. 

I only added the default one because qdrant did a good job of uploading the onnx without a pooling config. ^^;

Down side is more code, more behaviour, more maintainance. Happy to remove it if you feel it actually make the code harder to maintain.

Reply:
Right now the switch in `get_default_pooling_method()` doesn't have a catch all clause. So every model must be specified.
So, I guess, reading the config file won't be necessary and we can specify it when adding the model.

Reply:
no worries, it's gone now.